Cargo.toml

```
@@ -27,16 +27,16 @@ soroban-builtin-sdk-macros = { version = "=20.0.0-rc2", path = "soroban-builtin-
 
 # NB: When updating, also update the version in rs-soroban-env dev-dependencies
 [workspace.dependencies.stellar-xdr]
-version = "=20.0.0-rc1"
+version = "20.0.0"
```

Do you want to keep this pinned to v20.0.0 exactly? Or okay if we let env run with v20.0.1, or v20.1.0?
```suggestion
version = "=20.0.0"
```